cfgov/ask_cfpb/scripts/migrate_knowledgebase.py

```diff
@@ -142,9 +142,49 @@ def fix_tips(answer_text):
 
 
 def get_or_create_landing_pages():
-    """Create English and Spanish landing pages"""
+    """
+    Create two Spanish and one English landing pages.
+
+    Spanish gets two parents:
+    - `/es/`, as a placeholder for a future Spanish home page in Wagtail
+    - `/obtener-respuestas/`, the landing page for Spanish ask-cfpb
+    """
+
     from v1.models import CFGOVPage
-    parent = CFGOVPage.objects.get(slug='cfgov').specific
+    root = CFGOVPage.objects.get(slug='cfgov').specific
+
+    def get_or_create_es_parent_page():
+        """Create the Spanish /es/ parent placeholder page"""
+        text_intro_stream_value = [
+            {'type': 'text_introduction',
+             'value': {
+                 'heading': 'Placeholder for future Spanish home page',
+                 'links': []}}]
+        # this unserved archived page is squatting on the /es/ slug
+        dinero = CFGOVPage.objects.get(id=51)
```

Comment:
I think I see now. Would a better approach be to modify `get_or_create_page` to optionally look under a particular parent? Relying on `id` 51 to be there seems brittle.